lib/deploy/events/apiGateway/methods.js

```diff
@@ -236,6 +236,22 @@ module.exports = {
     return defaultTemplate;
   },
 
+  getIntegrationResponseTemplates(StatusCode, http) {
+    const defaultTemplate = {};
+
+    if (_.has(http, "response.statusCodes")) {
```

Comment:
how about `if(_.has(http, `response.statusCodes.${StatusCode}.template`))`? or 
```
const template = _.get(http, `response.statusCodes.${StatusCode}.template`);
if (template) {
  return Object.assign(defaultTemplate, {'application/json': template});
} else {
  return defaultTemplate;
}
```

Reply:
Sure. I have never programmed in node, so I am trying to figure out how to improve on codebase (or write test cases for that matter)